### PR TITLE
Rpcz, Statsz, Tracez, and Traceconfigz page styling modifications

### PR DESCRIPTION
**Pull request by adamgarza**
OpenCensus logo & color scheme have been introduced across all Zpages. Also, table views have been standardized across all pages.

### CODE REVIEW

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/RpczZPageHandler.java [hunk 1]
```diff
@@ -233,6 +233,11 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     out.write("<meta charset=\"utf-8\">\n");
     out.write("<title>RpcZ</title>\n");
     out.write("<link rel=\"shortcut icon\" href=\"//www.opencensus.io/favicon.ico\"/>\n");
+    out.write(
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/RpczZPageHandler.java]: Consider putting the style code into another static method (like `emitStyle()`), so that we can find and update the styles more quickly in the future.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java [hunk 2]
```diff
@@ -131,6 +130,11 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     out.write("<meta charset=\"utf-8\">\n");
     out.write("<title>StatsZ</title>\n");
     out.write("<link rel=\"shortcut icon\" href=\"//www.opencensus.io/favicon.ico\"/>\n");
+    out.write(
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java]: Please move this part to `emitStyles()`.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java [hunk 3]
```diff
@@ -100,8 +100,17 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     out.write("<meta charset=\"utf-8\">\n");
     out.write("<title>TraceConfigZ</title>\n");
     out.write("<link rel=\"shortcut icon\" href=\"//www.opencensus.io/favicon.ico\"/>\n");
+    out.write(
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Same here.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TracezZPageHandler.java [hunk 4]
```diff
@@ -162,6 +162,11 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     out.write("<meta charset=\"utf-8\">\n");
     out.write("<title>TraceZ</title>\n");
     out.write("<link rel=\"shortcut icon\" href=\"//www.opencensus.io/favicon.ico\"/>\n");
+    out.write(
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TracezZPageHandler.java]: ditto

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java (hunk 2 above)
Comment (bogdandrutu) [reply contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java]: Consider a separate .css file that we embed into the binary. This way we can share the same file between multiple implementations (languages).

Comment (songy23) [reply contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java]: Yes that sounds better, though I don't know how difficult it is to apply the same css file to all pages, since each zpage has its own class/id names. @adamgarza what do you think?

File: examples/src/main/java/io/opencensus/examples/zpages/ZPagesTester.java [hunk 5]
```diff
@@ -105,4 +105,4 @@ public static void main(String[] args) throws Exception {
     ZPageHandlers.startHttpServerAndRegisterAll(8080);
     recordExampleData();
   }
-}
+}
```
Comment (songy23) [inline examples/src/main/java/io/opencensus/examples/zpages/ZPagesTester.java]: I think the style error is from here - no new line at EOF. Could you revert this change?

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/RpczZPageHandler.java [hunk 6]
```diff
@@ -224,6 +224,32 @@ public String getUrlPath() {
     return RPCZ_URL;
   }
 
+  public static void emitStyle(PrintWriter out) {
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/RpczZPageHandler.java]: `emitStyle()` should be private.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java [hunk 7]
```diff
@@ -90,6 +90,32 @@ public String getUrlPath() {
     return TRACE_CONFIGZ_URL;
   }
 
+  public static void emitStyle(PrintWriter out) {
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Same here.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TracezZPageHandler.java [hunk 8]
```diff
@@ -153,6 +153,33 @@ public String getUrlPath() {
     return TRACEZ_URL;
   }
 
+  public static void emitStyle(PrintWriter out) {
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TracezZPageHandler.java]: ditto
